atlite/resources/windturbine/2016CACost_NREL_Reference_10MW_205_offshore.yaml

```diff
@@ -0,0 +1,19 @@
+# SPDX-FileCopyrightText: 2022 The Atlite Authors
+#
+# SPDX-License-Identifier: CC-BY-4.0
+
+# Turbine characteristic data
+name: 2016CACost_NREL_Reference_10MW_205
+# The manufacturer of this turbine
+manufacturer: NREL
+# Link to the original datasheet of the turbine
+source: https://nrel.github.io/turbine-models/2016CACost_NREL_Reference_10MW_205.html
+
+# Hub height in meters
+HUB_HEIGHT: 205.0
```

Comment:
`HUB_HEIGHT` seems incorrect for most (all?) turbines. For this turbine it would be `125 m` according to the linked source. The `205 m` are the rotor diameter.

Reply:
Yes, you are absolutely right! I assumed the rotor diameter as the hub height from the file name, now I've sourced it from the .rst in their docs.